[0.028s] Invoking command in '/home/mechax/zyb/radar_station/src/my_msgss/build/my_msgss': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/opt/ros/humble /usr/bin/cmake /home/mechax/zyb/radar_station/src/my_msgss -DCMAKE_INSTALL_PREFIX=/home/mechax/zyb/radar_station/src/my_msgss/install/my_msgss
[0.035s] -- Found ament_cmake: 1.3.6 (/opt/ros/humble/share/ament_cmake/cmake)
[0.116s] -- Found rosidl_default_generators: 1.2.0 (/opt/ros/humble/share/rosidl_default_generators/cmake)
[0.117s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.119s] -- Found rosidl_adapter: 3.1.5 (/opt/ros/humble/share/rosidl_adapter/cmake)
[0.136s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[0.252s] [31mCMake Error at /opt/ros/humble/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:59 (message):
[0.253s]   execute_process(/usr/bin/python3.10 -m rosidl_adapter --package-name
[0.253s]   my_msgss --arguments-file
[0.253s]   /home/mechax/zyb/radar_station/src/my_msgss/build/my_msgss/rosidl_adapter__arguments__my_msgss.json
[0.253s]   --output-dir
[0.253s]   /home/mechax/zyb/radar_station/src/my_msgss/build/my_msgss/rosidl_adapter/my_msgss
[0.253s]   --output-file
[0.253s]   /home/mechax/zyb/radar_station/src/my_msgss/build/my_msgss/rosidl_adapter/my_msgss.idls)
[0.253s]   returned error code 1:
[0.253s] 
[0.253s]   Error processing 'dist_point[] data' of 'my_msgss/Distpoints':
[0.253s]   ''dist_point' is an invalid message name.  It should have the pattern
[0.253s]   '^[A-Z][A-Za-z0-9]*$''
[0.253s] 
[0.253s]   Traceback (most recent call last):
[0.253s] 
[0.253s]     File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
[0.253s]       return _run_code(code, main_globals, None,
[0.253s]     File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
[0.253s]       exec(code, run_globals)
[0.253s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__main__.py", line 19, in <module>
[0.253s]       sys.exit(main())
[0.253s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/main.py", line 53, in main
[0.253s]       abs_idl_file = convert_to_idl(
[0.253s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__init__.py", line 19, in convert_to_idl
[0.253s]       return convert_msg_to_idl(
[0.253s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/msg/__init__.py", line 28, in convert_msg_to_idl
[0.253s]       msg = parse_message_string(package_name, input_file.stem, content)
[0.253s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 520, in parse_message_string
[0.253s]       Type(type_string, context_package_name=pkg_name),
[0.253s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 277, in __init__
[0.253s]       super(Type, self).__init__(
[0.253s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 205, in __init__
[0.253s]       raise InvalidResourceName(
[0.253s] 
[0.253s]   rosidl_adapter.parser.InvalidResourceName: 'dist_point' is an invalid
[0.253s]   message name.  It should have the pattern '^[A-Z][A-Za-z0-9]*$'
[0.253s] 
[0.253s] Call Stack (most recent call first):
[0.253s]   /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[0.253s]   CMakeLists.txt:14 (rosidl_generate_interfaces)
[0.254s] 
[0.254s] [0m
[0.254s] -- Configuring incomplete, errors occurred!
[0.254s] See also "/home/mechax/zyb/radar_station/src/my_msgss/build/my_msgss/CMakeFiles/CMakeOutput.log".
[0.256s] Invoked command in '/home/mechax/zyb/radar_station/src/my_msgss/build/my_msgss' returned '1': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/opt/ros/humble /usr/bin/cmake /home/mechax/zyb/radar_station/src/my_msgss -DCMAKE_INSTALL_PREFIX=/home/mechax/zyb/radar_station/src/my_msgss/install/my_msgss
